estuary/utils/story.py

```diff
@@ -195,9 +197,51 @@ def get_wait_times(self, results):
 
             # Find the time between when the current artifact completes and the next one starts
             wait_time = next_artifact_start_time - completion_time
-            wait_times[index] = wait_time.seconds
+            wait_times[index] = wait_time.seconds + (wait_time.days * 86400)
+
+            # The 'wait time' between a FreshmakerEvent and a ContainerKojiBuild is still a part of
+            # the processing in a FreshmakerEvent, so we do not count it towards the total wait time
+            if artifact.__label__ != 'FreshmakerEvent':
+                total_wait_time += wait_time.seconds + (wait_time.days * 86400)
 
-        return wait_times
+        return (wait_times, total_wait_time)
+
+    def get_total_processing_time(self, results):
+        """
+        Get the total time spent processing in the pipeline.
+
+        :param list results: contains inflated results from Neo4j
+        :return: the seconds of total time spent processing
+        :rtype: int
+        """
+        len_story = len(results)
+        if len_story < 2:
+            raise RuntimeError('This function can\'t be called with one or zero elements')
```

Comment:
I don't think this restriction is necessary. The processing time can be calculated on a single node.